config/migrations/2023/20230925125154-remove-lpdc-data/20230925125154-delete-Address.sparql

```diff
@@ -0,0 +1,15 @@
+DELETE {
+  GRAPH ?g {
```

Comment:
This is removing too much, you'll see e.g. leidinggevenden an eredienstmandatarissen use these classes too.
-  best to scope all removals for the correct graph for the stuff in organization graphs 
- for the  public graph, scope it to the public graph and you will have to make a an explicit query, starting from conceptual public service and the walk the path until you find the exact resources to clean.

Reply:
Regarding the first point, `Address` and `ContactPoint` types are now scoped to LPDC organization graphs only; I checked other types that belong to the public graph, and all instances of those types were blank nodes generated by the `lpdc-ldes-consumer` (they can be safely deleted).

For the second point, it's not clear to me yet what needs to be done.

Reply:
About the second point, it's possible adressen were inserted from IPDC; from the conceptual public service.
The path would (approx) be conceptual-public-service->contactpoint->address. If that's the case these should be removed too. But don't remove all adressen you find in the public graph; only the ones attached to a conceptual public service.
(Please note that these check should occur before you cleaned the data)

Reply:
ah interesting; that might be related to yet to complete data coming from OP. To keep the cleansing scoped to exactly IPDC data; i would not delete these. But anyway; if understand correctly there is no address to delete in the public graph for ipdc rright?

Reply:
There are addresses in LPDC graphs. The full list of graphs containing address types from a recent production dump (October 18):
* http://mu.semte.ch/graphs/public
* http://mu.semte.ch/graphs/organizations/*/LoketLB-LPDCGebruiker
* http://mu.semte.ch/graphs/organizations/*/LoketLB-leidinggevendenGebruiker
* http://mu.semte.ch/graphs/organizations/*/LoketLB-eredienstBedienaarGebruiker
* http://mu.semte.ch/graphs/organizations/*/LoketLB-eredienstMandaatGebruiker
* http://redpencil.data.gift/id/deltas/producer/loket-leidinggevenden-producer
* http://mu.semte.ch/graphs/delta-producer-publication-maintainer/folding/deletes/dd0c9af0-531a-11ee-9586-79d37f8c0b12

They were obtained by skimming the results of this query:
```
SELECT DISTINCT ?g WHERE {
  GRAPH ?g {
    ?s a <http://www.w3.org/ns/locn#Address> ;
      ?p ?o .
  }
}
```

The `delete-address` migration file is scoped to LPDC graphs only, and does not touch any other graph content.

Reply:
thanks. Can you check to what these Adressen are connected in the public-graph? (it might be stuff coming from IPDC)

Reply:
If I query what subjects reference these vestigingen (`?x ?y ?vestiging), I get no results; it seems there's nothing that points to them.